spec/features/admin/content_spec.rb

```diff
@@ -18,23 +18,36 @@
     fill_in 'footer_twitter_url', with: 'http://twitter.com/me'
     fill_in 'footer_links_md', with: '[markdown link](/)'
     click_button 'Update'
-    page.should have_content 'Your content has been successfully updated!'
+    expect(page).to have_content 'Your content has been successfully updated!'
 
     visit root_path
 
     # Then social media icons are only shown if they have a value
-    page.should_not have_selector 'i.ofn-i_044-facebook'
-    page.should     have_selector 'i.ofn-i_041-twitter'
+    expect(page).not_to have_selector 'i.ofn-i_044-facebook'
+    expect(page).to     have_selector 'i.ofn-i_041-twitter'
 
     # And markdown is rendered
-    page.should have_link 'markdown link'
+    expect(page).to have_link 'markdown link'
   end
 
   scenario "uploading logos" do
     attach_file 'logo', "#{Rails.root}/app/assets/images/logo-white.png"
     click_button 'Update'
-    page.should have_content 'Your content has been successfully updated!'
+    expect(page).to have_content 'Your content has been successfully updated!'
 
-    ContentConfig.logo.to_s.should include "logo-white"
+    expect(ContentConfig.logo.to_s).to include "logo-white"
+  end
+
+  scenario "setting user guide link" do
+    fill_in 'user_guide_link', with: 'http://www.openfoodnetwork.org/platform/user-guide/'
+    click_button 'Update'
+
+    expect(page).to have_content 'Your content has been successfully updated!'
+    expect(ContentConfig.user_guide_link).to eq 'http://www.openfoodnetwork.org/platform/user-guide/'
```

Comment:
being this a feature spec, this expectation shouldn't be here because it belongs to a deeper testing level (unit? something in between?). It does not exert the UI but a class. If line 50 passes, is because `ContentConfig` worked. I would simply remove it because `Spree::Preferences::FileConfiguration` is already tested by Spree.